xml_inscriptions/bib_only/MA.Bos.MFA.G.01.7479.xml

```diff
@@ -35,13 +35,78 @@
                   </msItem>
                </msContents>
-               <physDesc><!--objectDesc, handDesc, decoDesc go here--></physDesc>
-               <history><!--summary, origin, provenance go here--></history>
+               <!--  *************************************  <physDesc>  ********************************* -->
+               <physDesc>
+                  
+                  <!--  *************************************  <objectDesc>  *********************************  -->
+                  
+                  <objectDesc ana="#fragment #object_other">
+                     <supportDesc ana="#metal.bronze">
+                        <support>
+                           <p> bronze helmet </p>
+                           <dimensions type="surface" unit="cm">
+                              <height>25</height>
+                              <width>25</width>
+                              <depth/>
+                           </dimensions>
+                        </support>
+                        <condition ana="#complete.broken">
+                           <p> </p>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="0" writtenLines="1">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#inscribed"> "incised" </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p/>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               
+               <!--  *************************************  <history>  *********************************  -->
+               
+               <history>
+                  
+                  <summary>
+                     <rs/>  <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                  </summary>
+                  
+                  <origin>
+                     <date evidence="#unspecified" notBefore="-0549" notAfter="-0526">550-525
+                        BC</date>
+                     <placeName ref="europe.greece"/>
+                  </origin>
+                  <provenance type="transferred" subtype="purchased">
+                     <p>purchased by MFA from Edward Perry Warren, December 1901 <date when="1901">1901</date></p>
```

Comment:
You can surround the various parties within each `<provenance>` element with a `<persName>` or `<orgName>`; you can further indicate what their role in the transaction was by listing something like `<persName role="donor">`, for instance.